src/components/MyPage/Edit/Content/index.tsx

```diff
@@ -87,10 +97,16 @@ const Content = ({ profile }: Props) => {
       formData.append('profileImg', image);
     }
 
-    editProfile(formData).then((res) => {
-      console.log(res);
-      router.push('/mypage');
-    });
+    editProfile(formData)
+      .then((res) => {
+        console.log(res);
+        setIsSumitSuccess(true);
```

Comment:
위에 오타 고치면서 여기도 수정해야 할 것 같습니다!
```suggestion
        setIsSubmitSuccess(true);
```

Reply:
오마이갓 감사해요.. 고치겠습니다